src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReferenceConfidenceModel.java

```diff
@@ -401,7 +401,8 @@ public ReferenceConfidenceResult calcGenotypeLikelihoodsOfRefVsAny(final int plo
                                                                        final byte refBase,
                                                                        final byte minBaseQual,
                                                                        final MathUtils.RunningAverage hqSoftClips,
-                                                                       final boolean readsWereRealigned) {
+                                                                       final boolean readsWereRealigned,
+                                                                       final double activeRegionAltMultiplier) {
```

Comment:
rename the new parameter in this method, since this method can be (and is) called by other parts of the code that are not related to active region detections.  maybe `altReadWeight` instead?